src/Artifacts.jl

```diff
@@ -6,14 +6,27 @@ import ClimaUtilities.ClimaArtifacts: @clima_artifact
 
 import LazyArtifacts
 
+"""
+     saturated_land_ic_path(; context = nothing)
+
+Returns the path to the spun-up land initial conditions created with zero
+flux at the bottom of a 15m soil column; these boundary conditions allow
+for a water table to form.
```

Comment:
I wonder if it would be possible/helpful to include in the repo a setup that is compatible with running from or generating these ICs